src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.AspNetCore.Authorization;
 #endif
 using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
```

Comment:
This comes out of some previous API review decisions. We wanted the builder APIs for JS components to move into `Microsoft.AspNetCore.Components.Web`, and so we now need references to that namespace in the project templates. The WebAssembly one already has it, so it just needs to be added here.